Gemfile

```diff
@@ -5,12 +5,18 @@ gemspec
 
 group :development do
 	gem 'pry'
-	gem 'pry-coolline'
```

Comment:
As a reference, the `Gemfile` is using tab as intent. But it is not usual. It is invalid for my `rubocop` check.
So, I used 2 spaces as indent in the file.